less/home-page.less

```
@@ -21,11 +19,32 @@
         border-radius: 10px;
       }
     }
+
+    @media @mobile{
```

Your images do not span the width of their container. You can do this by nesting in another layer of specificity. Like this:

```suggestion
    @media @mobile{
         flex-direction: column;
  
   .text-content, .img-content{
        width: 100%;
                 img {
                         width: 100%;
                        }
    }
   }


```